openfisca_uk/parameters/contrib/ubi_center/basic_income/exempt_pensioners_from_tax_changes.yaml

```diff
@@ -0,0 +1,5 @@
+description: Exempt individuals over State Pension age from all reforms to Income Tax rates and thresholds.
+values:
+  2010-01-01: false
+metadata:
+  label: Exempt pensioners from Income Tax reforms
```

Comment:
Is this specifically earned income tax reforms?

Reply:
Yes, just the main rates and thresholds for labour income - updated the description